addons/portal/views/portal_templates.xml

```diff
@@ -73,7 +73,7 @@
                 <t t-if="_avatar">
                     <t t-if="user_id.image_256" t-set="avatar_source" t-value="image_data_uri(user_id.image_256)"/>
                     <t t-else="" t-set="avatar_source" t-value="'/web/static/src/img/placeholder.png'"/>
-                    <img t-att-src="avatar_source" t-attf-class="rounded-circle #{_avatar_class}" width="24" height="24" alt="" loading="eager"/>
+                    <img t-att-src="avatar_source" t-attf-class="rounded-circle portal-user-dropdown-avatar #{_avatar_class}" width="24" height="24" alt="" loading="eager"/>
```

Comment:
Thank you for the update. We're getting there, sorry for another comment.

Now that I think about it, I think we already had `o_object_fit_cover` class in v14? In that case we could use it directly instead of creating a new one.